CHANGELOG.md

```diff
@@ -11,6 +11,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (PR #192).
 
 ### Changed
+- **Breaking**: change binary types encode/decode for Python 3
+  to support working with varbinary (PR #211, #105).
+  Python 2 connector behavior remains the same.
+
+  Before this patch:
+
+  * encoding="utf-8" (default)
+
+    Python 3 -> Tarantool          -> Python 3
+    str      -> mp_str (string)    -> str
+    bytes    -> mp_str (string)    -> str
+                mp_bin (varbinary) -> bytes
```

Comment:
It poorly rendered on GitHub. I think you can use the same (nice!) formatting as you're used in the pull request description.

Reply:
Sorry, I forgot to check the compiled version of CHANGELOG. Fixed!